nexus/src/app/background/tasks/physical_disk_adoption.rs

```diff
@@ -138,7 +139,9 @@ impl BackgroundTask for PhysicalDiskAdoption {
                 let zpool = Zpool::new(
                     Uuid::new_v4(),
                     inv_disk.sled_id.into_untyped_uuid(),
-                    disk.id()
+                    disk.id(),
+                    // See oxidecomputer/omicron#7875 for the 250G determination
+                    ByteCount::from_gibibytes_u32(250).into(),
```

Comment:
Could we pull this out into a named constant? It's also used in `nexus/src/app/rack.rs`, I'd like to define this value once

Reply:
definitely, good idea - done in cc38e26d7